package.json

```diff
@@ -17,15 +17,16 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "@types/lodash": "^4.14.149",
     "@types/node": "^12.12.6",
     "typescript": "^3.7.2"
   },
   "dependencies": {
-    "@types/lodash": "^4.14.147",
     "discord.js": "^11.5.1",
     "dotenv": "^8.2.0",
     "lodash": "^4.17.15",
     "request": "^2.88.0",
+    "request-promise": "^4.2.5",
     "ts-node-dev": "^1.0.0-pre.44"
```

Comment:
ts-node-dev, dotenv should both be devDependencies. You could remove these dependencies and still run your program in production with regular node and injecting environment variables normally which is the usual flow